openid-connect-server/src/test/java/org/mitre/openid/connect/service/impl/TestMITREidDataService_1_1.java

```diff
@@ -1207,16 +1207,16 @@ public void testExportGrants() throws IOException {
                 } else {
                     assertThat(site.get("whitelistedSiteId").getAsLong(), equalTo(compare.getWhitelistedSite().getId()));
                 }
-                if (site.get("approvedAccessTokens").isJsonNull() || site.getAsJsonArray("approvedAccessTokens") == null) {
-                    assertTrue(compare.getApprovedAccessTokens() == null || compare.getApprovedAccessTokens().isEmpty());
-                } else {
-                    assertNotNull(compare.getApprovedAccessTokens());
-                    Set<String> tokenIds = new HashSet<String>();
-                    for(OAuth2AccessTokenEntity entity : compare.getApprovedAccessTokens()) {
-                        tokenIds.add(entity.getId().toString());
-                    }
-                    assertThat(jsonArrayToStringSet(site.getAsJsonArray("approvedAccessTokens")), equalTo(tokenIds));
-                }
+//                if (site.get("approvedAccessTokens").isJsonNull() || site.getAsJsonArray("approvedAccessTokens") == null) {
```

Comment:
We're going live in about a week and i think if we could have a proper release with this change it will be alot more acceptable (to non technical upper level management) rather than us having our own custom patched version, so having a 1.1.17 release with this change would be nice.  It's does fix what i would classify as a major defect which i think is deserving of a special release in 1.1.x.